DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -27,22 +27,13 @@
         "version" : "3.0.0"
       }
     },
-    {
```

Comment:
This branch has a reference to local BSK instead of branch `pete/pp-free-trials-core-implementation`